cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1629,6 +1619,61 @@ public void addSubscriptionTags(String[] tagIds) {
         addSubscriptionTagTrackingConsent(tagIds);
     }
 
+    public void addSubscriptionTopic(String topicId, CompletionListener completionListener) {
```

Comment:
the second parameter should still be optional and we need to check if its not null before calling it then, same for removeSubscriptionTopic(...)